scripts/fib.py

```diff
@@ -0,0 +1,10 @@
+def fib(n: int) -> int:
```

Comment:
Add a docstring for this

```suggestion
def fib(n: int) -> int:
    """Calculate the nth Fibonacci number."""
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)